lib/genome/groupers/drug_grouper.rb

```diff
@@ -1,154 +1,149 @@
 module Genome
   module Groupers
     class DrugGrouper
-      @alt_to_pubchem = Hash.new() {|hash, key| hash[key] = []}
-      @alt_to_other = Hash.new() {|hash, key| hash[key] = []}
-      @alt_to_pubchem_cid = Hash.new() {|hash, key| hash[key] = []}
 
-      def self.run
-        ActiveRecord::Base.transaction do
-          puts 'reset groups'
-          reset
-          puts 'preload'
-          preload
-          puts 'create groups'
-          create_groups
-          puts 'add members'
-          add_members
-          puts 'add drug aliases'
-          add_aliases
-          puts 'add drug attributes'
-          add_attributes
+      def run
+        begin
+          newly_added_claims_count = 0
+          drug_claims_not_in_groups.in_groups_of(1000, fill_with=false) do |claims|
```

Comment:
Ah, I see. I'm actually assigning a variable `fill_with` to value `false`, and that's getting passed in.

I understand what you were saying now. Good catch, definitely not my intent.